pytest.ini

```diff
@@ -0,0 +1,3 @@
+[pytest]
+filterwarnings =
+    ignore::DeprecationWarning
```

Comment:
Curious what led to this config?

Reply:
You get these warnings at the end of the pytest run that you can't really do anything about:

```
=============================== warnings summary ===============================
lambda/tests/test_new_account_iam_role.py::test_main_func_valid_arguments
  /home/kbalk/.pyenv/versions/3.8.6/lib/python3.8/site-packages/boto/plugin.py:40: DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
    import imp

-- Docs: https://docs.pytest.org/en/stable/warnings.html
=========================== short test summary info ============================
```

Reply:
Oh ok. Though, `boto` is the old aws python SDK. Weird. I see it coming from `moto`:

```
$ pipdeptree -r -p boto
boto==2.49.0
  - moto==1.3.16 [requires: boto>=2.36.0]
```

And, I see it was removed in `moto`, but that hasn't made it to a release yet, https://github.com/spulec/moto/pull/3503

Reply:
No changes needed.